.github/workflows/generate_matrix.sh

```diff
@@ -16,8 +16,9 @@
 set -e
```

Comment:
Should we replace this shell script with a Python script?

Reply:
Yes.

Reply:
Done